lib/galaxy/model/migrations/dbscript.py

```diff
@@ -24,31 +26,9 @@
 GXY_CONFIG_PREFIX = "GALAXY_CONFIG_"
 TSI_CONFIG_PREFIX = "GALAXY_INSTALL_CONFIG_"
 
-# Update this dict with tags for each new release.
-# Note: the key should NOT be a prefix of an existing revision hash in alembic/versions_gxy/.
-# For example, if we have a hash 231xxxxxxxxx and use 231 as the key for release 23.1,
-# then using 231 as a partial revision identifier like `sh manage_db.sh upgrade 231`
-# will map to release 23.1 instead of revision 231xxxxxxxxx.
-REVISION_TAGS = {
-    "release_22.01": "base",
-    "22.01": "base",
-    "release_22.05": "186d4835587b",
-    "22.05": "186d4835587b",
-    "release_23.0": "caa7742f7bca",
-    "23.0": "caa7742f7bca",
-    "release_23.1": "e93c5d0b47a9",
-    "23.1": "e93c5d0b47a9",
-    "release_23.2": "8a19186a6ee7",
-    "23.2": "8a19186a6ee7",
-    "release_24.0": "55f02fd8ab6c",
-    "24.0": "55f02fd8ab6c",
-    "release_24.1": "04288b6a5b25",
-    "24.1": "04288b6a5b25",
-    "release_24.2": "a4c3ef999ab5",
-    "24.2": "a4c3ef999ab5",
-    "release_25.0": "c716ee82337b",
-    "25.0": "c716ee82337b",
-}
+REVISION_FILE = os.path.join(os.path.dirname(__file__), "dbrevisions.yml")
```

Comment:
This doesn't work for non-filesystem based packages. Please use resource_string to load contents.